core/assets/bundles/bundle_de.properties

```diff
@@ -1139,7 +1146,7 @@ block.coal-centrifuge.description = Verfestigt Öl zu Kohlenstücken.
 block.incinerator.description = Vernichtet beliebige überschüssige Materialien oder Flüssigkeiten.
 block.power-void.description = Verschlingt den kompletten übrigen Strom. Nur im Sandkasten-Modus verfügbar.
 block.power-source.description = Erzeugt unendlich viel Strom. Nur im Sandkasten-Modus verfügbar.
-block.item-source.description = Produziert unendlich items. Nur im Sandkasten-Modus verfügbar.
+block.item-source.description = Schürft unendlich Ressourcen. Nur im Sandkasten-Modus verfügbar.
```

Comment:
Schuerft ist ein wages Wort, da man diesen block unabhaengig von erzen bauen kann. Produziert kann eigentlich so bleiben

Reply:
Produzieren ist eine handwerkliche oder maschinelle Tätigkeit die Ressourcen als Input benötigen, Während schüften explizit das Abbauen von Erzen meint.
Alternativ könnte man auch "Erzeugt" verwenden. Es ist genereller und definiert nicht die angewendete Methode.

Reply:
Produziert wird aber auch als solche Bedeutung in der englischen version genutzt, der block erstellt ja ressourcen aus der luft.
Erzeugen waere aehnlich, Generieren ist glaube das wort was wir suchen